src/components/SetUserProperties.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div></div>
+</template>
+
+<script>
+import axios from "axios";
+
+export default {
+  methods: {
+    saveLocalUserConfigs() {
+      axios
+        .get(
+          process.env.VUE_APP_BACKEND +
+            process.env.VUE_APP_BACKEND_USER_CONFIG +
+            "?user-id=" +
+            this.$store.getters.getUserId
+        )
+        .then((response) => {
+          this.$store.dispatch("saveConfigs", {
+            configs: JSON.stringify(response.data), // save user config locally
+          });
+        });
+    },
+
+    setLocale() {
```

Comment:
There is `UserProperties.setLocale` and `LocaleSwitcher.setLocale`. There is some cleanup here -- maybe move that method also here and split by functionality, since both are dealing with the user config?